arelle/plugin/validate/DBA/rules/th.py

```diff
@@ -16,6 +17,38 @@
 
 _: TypeGetText
 
+
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+)
+def rule_th01(
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation]:
+    """
+    DBA.TH01: First “schemaRef” must contain “http://archprod.service.eogs.dk/taxonomy/”
+    This rule doesn't check that the first schemaRef is the first element in the instance document.
+    However, combined with DBA.TH10 that errors if there is more than one schemaRef, it should be sufficient.
+    """
+    modelXbrl = val.modelXbrl
+    for doc in modelXbrl.urlDocs.values():
+        if doc.type == ModelDocument.Type.INSTANCE:
+            for refDoc, docRef in doc.referencesDocument.items():
+                if docRef.referringModelObject.localName == "schemaRef":
+                    href = refDoc.uri
+                    if href.startswith(pluginData.schemaRefUri):
+                        continue
+                    else:
+                        yield Validation.error(
+                            codes='DBA.TH01',
+                            msg=_('First "schemaRef" must contain "{}".'
```

Comment:
I know the validation spreadsheet calls out `first`, but I think based on how you have this validation working, you should remove that word in both sentences.